aspnetcore/blazor/tutorials/movie-database-app/part-7.md

```diff
@@ -0,0 +1,478 @@
+>
+> To delete all of the records in the database, use one of the following approaches:
+>
+> * Run the app and use the delete links in the browser. This approach is reasonably fast when there are only a few records to delete.
+> * Using database tooling&dagger;, delete the database records. In most database tools that permit direct manipulation of data, open the database table's data view in the tooling. When the table opens to show the movie records, select the first record. Hold the <kbd>Shift</kbd> key and select the last record to select all of the records in the database. With all of the records selected, press the <kbd>Delete</kbd> key to delete the records. You may need to adjust these instructions depending on the database tooling in use. Consult the tool's documentation for more information. This approach is reasonably fast when there are many records to delete.
+> * Using database tooling&dagger;, execute a SQL query against the movie table to delete the table's records. The SQL command is `DELETE FROM dbo.Movie;`. You can save the query for later use if you wish. Otherwise, close the file without saving it. Consult the tool's documentation for the exact steps. This approach is especially useful when the table is large and contains hundreds to thousands of records.
+>
+> &dagger;*database tooling*: There are many free and retail database tools available on the open market that can work with SQL databases. Consult Internet resources to find one suitable for your database and platform.
+>
+> **CAUTION**: *Always use extreme caution when deleting records. Deleting records is permanent without taking additional data loss mitigation steps. Production databases often provision automatic backup copies of data, either instantaneously as the database is modified or periodically, including with off-site copies and permanent physical storage of data.*
+>
+> After deleting all of the records, run the app. The initializer reseeds the database and includes the correct movie ratings for the `Rating` field based on the seeding code.
+
+## Drop and recreate the database for non-SQL Server providers
+
+**Skip this section if you successfully migrated the database using the preceding guidance.**
+
+In this tutorial, EF Core migrations are used when possible. A migration updates the database schema to match changes in the data model. However, migrations can only make changes to the database that the EF Core provider supports. While the SQL Server provider has wide support for migration tasks, other provider's capabilities are limited. For example, support may exist for adding a column (the `ef migrations add` command succeeds), but support may not exist for removing or changing a column (the `ef database update` command fails). Due to these limitations, you can drop and recreate the database using the guidance in this section.
```

Comment:
This could be addressed with a heading update. However, it seems that the ***entire section*** can be 🔪 because the only reason that Rick had all of this in the RP/MVC tutorials was to deal with the migration not working for SQLite.